### ISSUE TITLE
Module not found: Can't resolve 'path' on NextJS `edge` runtime.

### ISSUE THREAD

**Issue by homanp** [hwchase17/langchainjs#994]
## Summary
When running langchainjs on the `edge` runtime in NextJS I see the following exception:
```
Error: Module not found: Can't resolve 'path'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/langchain/dist/base_language/count_tokens.js
./node_modules/langchain/dist/llms/openai.js
./node_modules/langchain/llms/openai.js
```

Currently running version `0.0.63`.

**Comment by nfcampos** [comment]
See https://github.com/hwchase17/langchainjs/tree/main/test-exports-vercel for a working example

**Comment by homanp** [comment]
yep, I use the Next13 `appDir` route handlers. Perhaps that env differs somehow. 

https://beta.nextjs.org/docs/routing/route-handlers

**Comment by mushan0x0** [comment]
Me too.

**Comment by Nedomas** [comment]
I believe it's because of this line:
https://github.com/hwchase17/langchainjs/blob/main/langchain/src/base_language/count_tokens.ts#L62

Not sure why - maybe Next.js 13 app dir route handlers with `runtime: 'edge'` import files differently?

Commenting this line fixes the error. 

@nfcampos I understand it is intended that tiktoken will not be used in edge/browser (and be allowed to error here)?

**Comment by nfcampos** [comment]
Ah interesting, thanks for investigating!

@dqbd any thoughts on this one? Would it be fixed/fixable with https://github.com/hwchase17/langchainjs/pull/847

**Comment by dqbd** [comment]
Hello! That definitely sounds like an issue with @dqbd/tiktoken, will address it as soon as possible

**Comment by nfcampos** [comment]
Cool, thank you!
